manifests/b/BastianPedersen/lazydocker/0.23.1/BastianPedersen.lazydocker.installer.yaml

```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: BastianPedersen.lazydocker
+PackageVersion: 0.23.1
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: lazydocker.exe
+  PortableCommandAlias: lazydocker

```

Comment:
```suggestion
  PortableCommandAlias: lazydocker
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Docker.DockerDesktop
```

Reply:
Thank you very much! I was unaware that this option existed.